### ISSUE TITLE
[Enhancement ] YouTube video on chat

### ISSUE THREAD

**Issue by ThaerM** [egbakou/WhatsAppUI#2]
First Thank you for the great UI.
Can we add some enhancement like video link from YouTube. (show the video on view movable).

**Comment by egbakou** [comment]
You're welcome.
I didn't fully understand your issue.
Do you mean demo of the UI ?

**Comment by ThaerM** [comment]
No, i mean enhancement like when you receive video on Whats App you can open it on pop up like this video https://youtu.be/5sKiyK0CIb0?t=125

**Comment by egbakou** [comment]
Ah okay i see.
